### ISSUE TITLE
[Jetpack Content Migration Flow] Fix accessibility issues with large font size on migration steps

### ISSUE THREAD

**Issue by ovitrif** [wordpress-mobile/WordPress-Android#17467]
This task is for fixing accessibility -- large font sizes -- issues on the following screens of the WordPress to Jetpack migration flow:

- Notifications Screen
- Success Screen

The content of the migration screens should be scrollable vertically, in order to make it possible for all users to see the content, also when they have the largest font size set on their device.

Currently if the font size is set to max via accessibility settings on the device, part of the content on the Notifications and Success screens is not visible, and it's not possible to scroll in order to see and read the text.
